feature-libs/asm/components/index.ts

```diff
@@ -4,6 +4,7 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+export * from './asm-bind-cart/asm-bind-cart.component';
```

Comment:
Should we also export the dot spinner?

Is the dot spinner reusable enough so we add it in core instead of ASM?  Or perhaps it's tailored tightly to the ASM UI.

Reply:
I think it is expected to part of the ASM module's style, though it makes sense that the ASM component module export it.